definitions/divorce/json/FixedLists/FixedLists.json

```diff
@@ -1266,6 +1266,234 @@
     "ListElement": "Cheque",
     "DisplayOrder": 4
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.